src/MusicalScore/ScoreIO/InstrumentReader.ts

```diff
@@ -385,11 +375,20 @@ export class InstrumentReader {
             this.currentStaffEntry.Timestamp.Equals(new Fraction(0, 1)) && !this.currentStaffEntry.hasNotes()
           );
           this.saveAbstractInstructionList(this.instrument.Staves.length, beginOfMeasure);
-          if (this.openChordSymbolContainers.length !== 0) {
-            this.currentStaffEntry.ChordContainers = this.openChordSymbolContainers;
-            // TODO handle multiple chords on one note/staffentry
-            this.openChordSymbolContainers = [];
-          }
+          //TODO: this if block handles harmony/chords on the next note/staffentry element, so it assumes that a
+          //harmony is given before the staff entry, but when a harmony is given after a staff entry element with a backup node
+          //it is put on the next note/staffentry and the last chord item is never parsed at all.
+          //this means
+          //Ik moet eerst beter begrijpen wanneer currentStaffEntry geset wordt.
```

Comment:
;) I understand because I speak a bit of Plattdeutsch, but maybe this comment should be translated or transferred to another branch where you work on this issue.
It's definitely not a bad idea to leave a TODO comment here and work on it in another branch later, but it should be complete and completely in English ;)

Reply:
Sorry, I should have deleted all console logs and dutch comments. 
I use dutch comments as a reminder that they are only temporary and should have been deleted before commiting.